.gitignore

```diff
@@ -34,3 +34,4 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.idea/LunchWebApp.iml
```

Comment:
Again, please add empty line in the end of file, because of convection. Even git says about it:)
![image](https://user-images.githubusercontent.com/54136347/124726224-3c5e0000-df16-11eb-9d3a-b2d2b707e2a8.png)